cpp/src/barretenberg/stdlib/hash/keccak/keccak.hpp

```diff
@@ -5,7 +5,7 @@
 #include "barretenberg/stdlib/primitives/byte_array/byte_array.hpp"
 
 namespace proof_system::plonk {
```

Comment:
I suppose this isnt the time/place but all of these stdlib objects should be pulled out of the plonk namespace correct?

Reply:
Yes, there is no reason for them to be in plonk